### PR TITLE
test shenyu-admin :add CommonUpstream Util Test.

### PR DESCRIPTION
**Pull request by chenhao26-nineteen**
add CommonUpstreamUtils Test

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Common upstream utils test.
+ * @ClassName: CommonUpstreamUtilsTest
+ * @Author: ChenHao26
+ * @Date: 2022/10/27 16:13
+ */
+public class CommonUpstreamUtilsTest {
```
Comment (yu199195) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java:37]: add final

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Common upstream utils test.
+ * @ClassName: CommonUpstreamUtilsTest
+ * @Author: ChenHao26
```
Comment (yu199195) [inline shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java:34]: remove all note

Comment (chenhao26-nineteen) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java:34]: ok i will fix it

File: shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java (hunk 1 above)
Comment (chenhao26-nineteen) [reply shenyu-admin/src/test/java/org/apache/shenyu/admin/utils/CommonUpstreamUtilsTest.java:37]: ok

**Review by li-keguo — APPROVED** [review]

**Pull request merged**